### PR TITLE
add isort to CI

### PR DESCRIPTION
**Pull request by mathause**
Note that this should fail now, isort is not yet run.

### CODE REVIEW

File: .github/workflows/ci-cd-workflow.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ jobs:
       run: |
           which python
           make -B conda-environment
+          isort mesmer examples tests setup.py
```
Comment (mathause) [inline .github/workflows/ci-cd-workflow.yml:44]:
```suggestion
          isort  --check mesmer examples tests setup.py
```

File: .github/workflows/ci-cd-workflow.yml [hunk 2]
```diff
@@ -41,6 +41,7 @@ jobs:
       run: |
           which python
           make -B conda-environment
+          isort  --check mesmer examples tests setup.py
```
Comment (mathause) [inline .github/workflows/ci-cd-workflow.yml:44]:
```suggestion
          isort --check mesmer examples tests setup.py
```

**Comment by mathause** [comment]
Now it should pass...

**Review by leabeusch — APPROVED** [review]
Awesome, thanks @mathause! Especially for getting done this so quickly. I'll merge this into master now and am curious to learn what it'll do to #67 (ie will it switch to not passing the tests automatically right away? I guess I'll see in a little while ^^)

**Pull request merged**